### PR TITLE
feat: add Polish language support

### PR DESCRIPTION
**Pull request by nexthis**

### CODE REVIEW

**Review by kasperkristensen — DISMISSED** [review]
LGTM. Thanks for the PR @nexthis!

We will add a test suite that translations will need to pass. I am to have it ready by Monday, so will re-approve the PR then, if it passes said test.

**Note on Community Translations**
We greatly appreciate all PRs that help make Medusa accessible in more languages. However, we aren't able to validate that all translations are correct. Therefore, we rely on the community to ensure the translations are accurate and appropriate. If you see a translation that is incorrect, has missing keys, or has other issues, please open a PR to correct it.

**Comment by MrCasquette** [comment]
> LGTM. Thanks for the PR @nexthis!
> 
> We will add a test suite that translations will need to pass. I am to have it ready by Monday, so will re-approve the PR then, if it passes said test.
> 
> **Note on Community Translations** We greatly appreciate all PRs that help make Medusa accessible in more languages. However, we aren't able to validate that all translations are correct. Therefore, we rely on the community to ensure the translations are accurate and appropriate. If you see a translation that is incorrect, has missing keys, or has other issues, please open a PR to correct it.

Hello,
I've already mentioned it in another PR but you should use crowdin as it helps to manage many translation files, and so many features. Directus uses it, and that's awesome and easy for community to help translating missing keys. And it sync this with PR or someting similar. You should take a look, or even ask in the Directus Discord it's a cool community :)
Plus I believe there's a free version for open source project :)

As I'm not the user you should check. Hope it will helps.
Can't wait to have my french Dashboard :p

File: packages/admin/dashboard/src/i18n/translations/pl.json
Comment (kasperkristensen) [inline packages/admin/dashboard/src/i18n/translations/pl.json]:
```suggestion
    "$schema": "./$schema.json",
```

File: packages/admin/dashboard/src/i18n/translations/pl.json
Comment (kasperkristensen) [inline packages/admin/dashboard/src/i18n/translations/pl.json]: The following keys are missing or not part of the schema, and should be updated:

```
Validation failed for pl.json:
  Unexpected key: "items_few" at /general
  Unexpected key: "items_many" at /general
  Unexpected key: "numberOfKeys_few" at /json
  Unexpected key: "numberOfKeys_many" at /json
  Unexpected key: "header_few" at /json/drawer
  Unexpected key: "header_many" at /json/drawer
  Unexpected key: "numberOfKeys_few" at /metadata
  Unexpected key: "numberOfKeys_many" at /metadata
  Unexpected key: "count_few" at /dataGrid/errors
  Unexpected key: "count_many" at /dataGrid/errors
  Missing required key: "edit" at /products
  Unexpected key: "editProduct" at /products
  Unexpected key: "variantCount_few" at /products
  Unexpected key: "variantCount_many" at /products
  Unexpected key: "deleteWarning_few" at /products/media
  Unexpected key: "deleteWarning_many" at /products/media
  Unexpected key: "deleteWarningWithThumbnail_few" at /products/media
  Unexpected key: "deleteWarningWithThumbnail_many" at /products/media
  Unexpected key: "tableItem_few" at /products/variant
  Unexpected key: "tableItem_many" at /products/variant
  Unexpected key: "removeProductsWarning_few" at /collections
  Unexpected key: "removeProductsWarning_many" at /collections
  Unexpected key: "successToast_few" at /collections/products/add
  Unexpected key: "successToast_many" at /collections/products/add
  Unexpected key: "successToast_few" at /collections/products/remove
  Unexpected key: "successToast_many" at /collections/products/remove
  Unexpected key: "successToast_few" at /categories/products/add
  Unexpected key: "successToast_many" at /categories/products/add
  Unexpected key: "confirmation_few" at /categories/products/remove
  Unexpected key: "confirmation_many" at /categories/products/remove
  Unexpected key: "successToast_few" at /categories/products/remove
  Unexpected key: "successToast_many" at /categories/products/remove
  Missing required key: "removed" at /customers/groups
  Unexpected key: "successToast_few" at /customerGroups/customers/add
  Unexpected key: "successToast_many" at /customerGroups/customers/add
  Unexpected key: "title_few" at /customerGroups/customers/remove
  Unexpected key: "title_many" at /customerGroups/customers/remove
  Unexpected key: "description_few" at /customerGroups/customers/remove
  Unexpected key: "description_many" at /customerGroups/customers/remove
  Unexpected key: "wrongQuantity_one" at /orders/fulfillment/error
  Unexpected key: "wrongQuantity_few" at /orders/fulfillment/error
  Unexpected key: "wrongQuantity_many" at /orders/fulfillment/error
  Unexpected key: "showMoreActivities_few" at /orders/activity
  Unexpected key: "showMoreActivities_many" at /orders/activity
  Unexpected key: "items_few" at /orders/activity/events/fulfillment
  Unexpected key: "items_many" at /orders/activity/events/fulfillment
  Unexpected key: "items_few" at /orders/activity/events/return
  Unexpected key: "items_many" at /orders/activity/events/return
  Unexpected key: "shipping_few" at /stockLocations/shippingOptions/fields/count
  Unexpected key: "shipping_many" at /stockLocations/shippingOptions/fields/count
  Unexpected key: "returns_few" at /stockLocations/shippingOptions/fields/count
  Unexpected key: "returns_many" at /stockLocations/shippingOptions/fields/count
  Unexpected key: "values_few" at /taxRegions/fields/targets
  Unexpected key: "values_many" at /taxRegions/fields/targets
  Unexpected key: "numberOfTargets_few" at /taxRegions/fields/targets
  Unexpected key: "numberOfTargets_many" at /taxRegions/fields/targets
  Unexpected key: "additionalValues_few" at /taxRegions/fields/targets
  Unexpected key: "additionalValues_many" at /taxRegions/fields/targets
  Unexpected key: "confirmation_few" at /priceLists/products/delete
  Unexpected key: "confirmation_many" at /priceLists/products/delete
  Unexpected key: "successToast_few" at /priceLists/products/delete
  Unexpected key: "successToast_many" at /priceLists/products/delete
  Unexpected key: "removeCurrencyWarning_few" at /store
  Unexpected key: "removeCurrencyWarning_many" at /store
  Unexpected key: "removeCountriesWarning_few" at /regions
  Unexpected key: "removeCountriesWarning_many" at /regions
  Unexpected key: "removeSalesChannelsWarning_few" at /locations
  Unexpected key: "removeSalesChannelsWarning_many" at /locations
  Unexpected key: "removeProductsWarning_few" at /salesChannels
  Unexpected key: "removeProductsWarning_many" at /salesChannels
  Unexpected key: "successToast_few" at /apiKeyManagement/salesChannels
  Unexpected key: "successToast_many" at /apiKeyManagement/salesChannels
  Unexpected key: "warningBatch_few" at /apiKeyManagement/removeSalesChannel
  Unexpected key: "warningBatch_many" at /apiKeyManagement/removeSalesChannel
  Unexpected key: "successToastBatch_few" at /apiKeyManagement/removeSalesChannel
  Unexpected key: "successToastBatch_many" at /apiKeyManagement/removeSalesChannel
  Unexpected key: "stepsCompletedLabel_few" at /workflowExecutions
  Unexpected key: "stepsCompletedLabel_many" at /workflowExecutions
  Unexpected key: "years_few" at /dateTime
  Unexpected key: "years_many" at /dateTime
  Unexpected key: "months_few" at /dateTime
  Unexpected key: "months_many" at /dateTime
  Unexpected key: "weeks_few" at /dateTime
  Unexpected key: "weeks_many" at /dateTime
  Unexpected key: "days_few" at /dateTime
  Unexpected key: "days_many" at /dateTime
  Unexpected key: "hours_few" at /dateTime
  Unexpected key: "hours_many" at /dateTime
  Unexpected key: "minutes_few" at /dateTime
  Unexpected key: "minutes_many" at /dateTime
  Unexpected key: "seconds_few" at /dateTime
  Unexpected key: "seconds_many" at /dateTime
  ```

File: packages/admin/dashboard/src/i18n/translations/pl.json
Comment (kasperkristensen) [reply packages/admin/dashboard/src/i18n/translations/pl.json]: Once you have pointed the file to the schema, your editor should be able to tell you all the keys that are unexpected or missing.

**Review by kasperkristensen — CHANGES_REQUESTED** [review]
Hi @nexthis

Thanks for the PR. I have added some comments with required changes.

File: packages/admin/dashboard/src/i18n/translations/pl.json
Comment (nexthis) [reply packages/admin/dashboard/src/i18n/translations/pl.json]: Thanks, I didn't notice that. The fix has been added

File: packages/admin/dashboard/src/i18n/translations/pl.json
Comment (nexthis) [reply packages/admin/dashboard/src/i18n/translations/pl.json]: I notice that many keys were omitted or changed, so I decided to do it again. I checked the correctness against $schema.json. I verified the translations and added fixes

**Review by kasperkristensen — COMMENTED** [review]
LGTM! Thanks @nexthis

**Review by kasperkristensen — APPROVED** [review]

**Pull request merged**
